# Automatically Generated in 2017-06-12 22:33:47.453203
# MESSAGE: CONFIG_PARAM_SET_INT
# Description:Set new config parameter int value
uint8 ID = 200
uint8 sysid
uint8 compid
uint8 config_id
uint8 param_id
int32 value
